var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -31,10 +31,10 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.3', 'a5f65570cd9107ec8a8ec88f17b31bb1')
```

Comment:
>  The error occurs when building something that depends_on("flex")

please open a separate issue for this problem and also state what that _something_ is, so that others can reproduce. Regardless, the change you have **should be reverted**, as it's not a solution, but only a workaround.